README.md

```diff
@@ -1,6 +1,6 @@
-![SendGrid Logo](https://github.com/sendgrid/sendgrid-python/raw/master/twilio_sendgrid_logo.png)
+![SendGrid Logo](https://github.com/sendgrid/sendgrid-python/raw/HEAD/twilio_sendgrid_logo.png)
```

Comment:
Can this be relative, too?

Reply:
Yes, we just need to add the logo file to this repo.